CONTRIBUTING.md

```diff
@@ -0,0 +1,21 @@
+# Awesome React App
+
+### Introduction
+First of all, thank you for considering to contribute! 🙏
+
+Everybody can help make this project a little more awesome, doesn't matter if it's a big or small contribution.
+Please make sure you follow the code of conduct, which is located in the repository master directory, in order for everyone to have a peaceful and progressive experience.
+Find out below, which steps are needed to help out exactly.
+
+### How to contribute
+Follow these simple steps:
+1. Have a look at the issue page of the repository [here](https://github.com/ifactory-solutions/awesome-react-app/issues).
+2. Choose an issue you would like to resolve/contribute.
+3. Leave a comment in the issue, stating that you are working on it.
+4. Fork the repository [here](https://github.com/ifactory-solutions/awesome-react-app/fork_select).
+5. Start making the changes to your forked repository.
+6. If you think you're done, submit a pull request (For help, click [here](https://help.github.com/articles/creating-a-pull-request/)).
+
+7. (Optional step) Go to the issue page which you resolved and leave a comment referencing the pull request.
```

Comment:
You could also add a link to help those who don't know how to reference an issue 😉

Reply:
I'll add that.